data/xapleaux/xapleaux missions.txt

```diff
@@ -0,0 +1,171 @@
+			choice
+				`	(Panic and punch it.)`
+				`	(Take a step back and a deep breath.)`
+					goto calm
+			`	As your fist wildly connects with the alien's carapace it recoils from you, though you get the sense it was more from shock than from any force you yourself actually managed to impart. Everything comes to a sudden halt as every last one of them looks straight at you - all of them frozen in place, in some cases mid-step, to see what happens next.`
+			`	You realize that, regardless of what you attempt, you are unlikely to achieve anything except by apology.`
+			choice
+				`	(Raise your hands peaceably.)`
+			`	Events resume.`
+			label calm
+			`	The assembled aliens spring into motion, carrying equipment and hoses towards your ships and proceeding to clamber more or less all over it, inspecting it for heat damage and refuelling it, hopefully with the right stuff.`
+#This is only one of various future possible paths to unlock access to them, but for now it's the only one with requirements already in-game.
+			branch automata
+				has "event: wanderers: exiles have drones"
+				has "event: remnant: sign studies complete"
+			`	When they are done the same alien approaches you again and holds out a weighty gemstone to you. It is milky and translucent, and a brilliant shade of pink; cut into an octagonal shape with layers of surface faceting it scatters diffuse pink patterns across the ground as the brilliance of the core shines down through it. The alien points at you, and then at the crystal, and mimes holding it out to you before pointing at itself. It then points at the tablet which shows two animated symbols: an empty box filling, and a blob spraying something onto a panel marked with a fire symbol.`
+			`	You realize the alien is telling you that you can present the crystal to request fuel and repairs.`
+			`	The alien points at you and makes those same two sharp whistles before, almost as an afterthought, nodding as well.`
+			choice
+				`	(Make two noises and nod.)`
+			`	It points at itself, makes two sharp whistles and nods again. It looks at a nearby alien who also makes two sharp whistles and nods, and before you know it every alien in the vicinity seems to be making two sharp whistles and nodding to each other in fascination.`
+			action
+				
+			`	When the cacophony has settled the first alien looks at you, holds up its upper arms peaceably, and points to your ship with one of it's middle arms. It would seem that, while you are certainly welcome here to some extent, for now your stay is firmly over.`
+				depart
+			label revisit
+			`	Upon disembarking your ship you make sure you have your pink crystal token to hand, and your ship is immediately set upon by the various aliens doing refuelling and maintenance with the rapid-paced manner of communication that seems to be typical for them.`
+			branch automata

```

Comment:
It looks like this is repeated above in line 88. In line 88, there doesn't seem to be a branch for `automata` in that conversation. Is the branch in line 88 therefore redundant?

Reply:
No, this handles repeat, and first time triggers differently. It also is structured for my benefit in a way that is easy for me to add other branches in later/soon.